python/hyperon/exts/agents/agent_base.py

```diff
@@ -144,7 +144,7 @@ def _init_metta(self):
         # the caller space is not directly accessible as a context,
         # except the case when _metta is set via get_agent_atom with parent MeTTa
         if self._include_paths is not None:
-            env_builder = Environment.custom_env(include_paths=self._include_paths)
+            env_builder = Environment.custom_env(include_paths=self._include_paths, config_dir="")
```

Comment:
@Necr0x0Der, @besSveta  I am not sure if agent should read configuration from file system or not, my opinion that probably not, but I reproduced previous behavior.